spec/policies/request_for_comment_policy_spec.rb

```diff
@@ -66,6 +96,23 @@
       end
     end
 
+    shared_examples 'grants access to everyone but the author' do
```

Comment:
I thought of "granting access" to be the verbal equivalent of `show?` returning true. IMO, this shared example group and its `it`-blocks should be mapped clearer to the `report?` policy. What do you think of this naming? (Same for the other new shared example group)
```suggestion
    shared_examples 'grants report permissions to everyone but the author' do
```

Reply:
Thank you. This is a good idea. I moved the `permissions(:report?) do` to the example as well.

https://github.com/openHPI/codeocean/pull/2946/commits/e4a563d4f9ea1fac1a9dd029c48192a03ef25ca0

Reply:
Please don't forget to rename the `it`-blocks accordingly :pray:

Reply:
I renamed the it statements.